src/apps/schema/src/app/views/SchemaEdit/FieldSettings/RelatedOptions/RelatedOptions.js

```diff
@@ -17,74 +18,79 @@ export default connect((state) => {
   };
 })(function RelatedOptions(props) {
   const [loading, setLoading] = useState(false);
-  const [selectedModel, setSelectedModel] = useState(
-    props.field.relatedModelZUID
-  );
-  const [selectedField, setSelectedField] = useState(
-    props.field.relatedFieldZUID
-  );
+  const [selectedModel, setSelectedModel] = useState(null);
+
+  const modelOptions = useMemo(() => {
+    return props.models.map((m) => {
+      return {
+        value: m.ZUID,
+        inputLabel: m.label,
+        component: m.label,
+      };
+    });
+  }, [props.models]);
 
   const fieldOptions = useMemo(() => {
     return props.fields
-      .filter((f) => f.contentModelZUID === selectedModel)
+      .filter((f) => f.contentModelZUID === props.field.relatedModelZUID)
```

Comment:
Why is this changing from state to props?

Reply:
The state value was
const [selectedModel, setSelectedModel] = useState(
    props.field.relatedModelZUID
  );
  
  So I removed the redundancy